manager/src/test/java/com/redhat/service/bridge/manager/utils/DatabaseManagerUtils.java

```diff
@@ -79,6 +84,11 @@ private void deleteAllBridges() {
         ids.forEach(x -> bridgeDAO.deleteById(x));
     }
 
+    private void deleteAllWork() {
+        List<String> ids = workDAO.getEntityManager().createQuery("select w.id from Work w", String.class).getResultList();
```

Comment:
there's a `delete` keyword in JPQL as well

Reply:
I followed the other deletions for consistency; but happy to review them all tomorrow.

Reply:
Updated (and other clean-up code).